src/app-layout/__tests__/common.test.tsx

```diff
@@ -3,7 +3,14 @@
 import React from 'react';
 import { act } from 'react-dom/test-utils';
 import { AppLayoutWrapper } from '../../../lib/components/test-utils/dom';
-import { describeEachAppLayout, isDrawerClosed, drawersConfigurations, renderComponent } from './utils';
+import {
+  describeEachAppLayout,
+  drawerWithoutLabels,
+  isDrawerClosed,
+  renderComponent,
+  singleDrawer,
+  singleDrawerOpen,
```

Comment:
Refactored all drawers configuration presets, made them typescript friendly and also removed duplicates